allennlp/data/token_indexers/spacy_indexer.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict, List
+
+from overrides import overrides
+from spacy.tokens import Token as SpacyToken
+import torch
+import numpy
+
+from allennlp.common.util import pad_sequence_to_length
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.data.tokenizers.token import Token
+from allennlp.data.token_indexers.token_indexer import TokenIndexer
+
+
+@TokenIndexer.register("spacy")
+class SpacyTokenIndexer(TokenIndexer[numpy.ndarray]):
```

Comment:
Might be nice to signal in the name that this is giving you a vector from spacy - `SpacyTokenIndexer` just says I get _something_ from spacy, not what I get, and vectors aren't the first thing I think of with spacy.